src/main/java/com/haxwell/apps/quizki/config/WebSecurityConfig.java

```diff
@@ -49,16 +48,16 @@ public PasswordEncoder passwordEncoder() {
     protected void configure(HttpSecurity http) throws Exception {
         http.httpBasic().authenticationEntryPoint(auth)
         	.and()
-        	.csrf()
```

Comment:
Just wondering what the reason was here.. whether the code works "better" after the antMatchers, than before.. or something.

Reply:
I think this was a mistake. I had removed the CSRF filter while trying to debug something and I put it back in a different place. Most of the examples I see online have it in the front of the chain but I can't find any docs that say it needs to be there so I'm not sure if it's better in one place or the other. It's disabled here and in order to use it we would need to handle CSRF tokens in the requests so it wouldn't matter for now. I think it would still work since it would block the request from getting thru if it was enabled. 

[https://docs.spring.io/spring-security/site/docs/3.2.0.CI-SNAPSHOT/reference/html/csrf.html](url)